python/lsst/fgcmcal/fgcmFitCycle.py

```diff
@@ -492,6 +519,18 @@ def _fgcmFitCycle(self, butler):
 
         self._checkDatasetsExist(butler)
 
+        # Set defaults on whether to output standards and zeropoints
+        self.outputStandards = self.config.outputStandardsBeforeFinalCycle
+        self.outputZeropoints = self.config.outputZeropointsBeforeFinalCycle
+        self.resetFitParameters = True
+
+        if self.config.maxIter == 0:
+            # This is the final fit cycle, so we do not want to reset fit
+            # parameters, and we always want to output standards and zeropoints
+            self.outputStandards = True
+            self.outputZeropoints = True
+            self.resetFitParameters = False
+
```

Comment:
See above.  I added the `isFinalCycle` config flag to avoid this confusion.  When the final cycle is triggered, all of the key stuff is output and then the `fgcmOutputProducts.py` will transform these into stack-compatible formats.